ubclaunchpad/umami
# Recipe Routes GET POST

Test procedure

- To create recipe you must first POST /user to create the default user to satisfy the FK constraint
- Test GET /recipe for all recipes matches the default recipe built
- Test searching by keyword with query pararm /recipe?keyword=something
  - Would like some help to see if this is correct


TODO: 

- Create the filtering query by vegetarian and vegan will probably be done in a separate PR
